style.css

```diff
@@ -1 +1,247 @@
-/*  Add your styling here */
+* { 
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+
+body {
+    background-color: #ffffff;
+    display: grid;
+    grid-template-columns: repeat (2, 1fr);
+    grid-template-rows: 150px 30px auto 100px;
+    grid-template-areas: 
+    "header header"
+    "nav2 nav2"
+    "section section"
+    "footer footer"
+}
+
+header {
+    grid-area: header;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #ffcfd2;
+    opacity: 0.75;
+}
+
+.headerpic img {
+    width: 100%;
+    height: 100%;
+    margin-left: 1rem;
+}
+
+.title {
+    padding: 0.6;
+    max-width: 13rem;
+    margin: 6rem 0 0 0;
+}
+
+.title p {
+    font-size: large;
+    color: rgb(246 6 202);
+    font-weight: 700;
+
+}
+
+.nav2 {
+    display: none;
+}
+
+section {
+    grid-area: section;
+    margin: 1rem 1rem;
+    gap: 1rem;
+    display: grid;
+    grid-template-rows: repeat (2, auto);
+}
+
+.hero {
+    margin: 1em;
+}
+
+.hero h1 {
+    margin: 0 0 1rem 0;
+}
+
+.mainpicture {
```

Comment:
With class names / IDs in CSS, we generally use a hypen to separate words. This just makes it easier to read, so `mainpicture` would just become `main-picture`.